FLEDGE_extended_PA_reporting.md

```diff
@@ -204,6 +204,8 @@ Where `signalBucket` and `signalValue` is a dictionary which consists of:
     * 6: indicates seller rejected bid because “Creative Filtered - Language Exclusions”
     * 7: indicates seller rejected bid because “Creative Filtered - Category Exclusions”
     * 8: indicates seller rejected bid because "Creative Filtered - Did Not Meet The K-anonymity Threshold"
+    * 9: indicates bid produced by `generateBid()` was rejected because it failed a currency check
+    * 10: indicates bid passed through or altered by `scoreAd()` was rejected because it failed a currency check
```

Comment:
I'm not sure I understand the circumstances when this could happen, could we include an example or two?

Reply:
9 happens if the bid returned by generateBid doesn't match what's specified by perBuyerCurrency.
10 happens if the bid returned or passed through by scoreAd is in a component auction and doesn't match sellerCurrency or the perBuyerCurrency required by the top-level auction.

Reply:
Anyway, I feel like a lot of your comments are asking me to repeat information, I am quite reluctant to do that since that's just asking for it to be different in different spots. So maybe the right answer is to link to section 3.6 here.

Reply:
Adding "(e.g. the bid returned by `generateBid()` doesn't match what's specified by `perBuyerCurrency`)" to the end of the sentence for 9 seems good.
Adding "(e.g. the bid returned or passed through by `scoreAd()` in a component auction doesn't match the `sellerCurrency` or the `perBuyerCurrency` required by the top-level auction.)" to the end of the sentence for 10 seems helpful.

Reply:
Done